app-admin/pwman3/pwman3-0.10.0-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite?"
+inherit distutils-r1
+
+DESCRIPTION="A lightweight password-manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io/pwman3/"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="mongodb mysql postgres +sqlite test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="
+	>=dev-python/cryptography-2.3[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.3.0[${PYTHON_USEDEP}]
+	"
+
+DEPEND="
+	${CDEPEND}
+	test? ( dev-python/pexpect[${PYTHON_USEDEP}] )
```

Comment:
Most likely a BDEPEND, which then makes whole `COMMON_DEPEND` useless.

Reply:
Not sure I understand your comment about COMMON_DEPEND here. Please see if this is correct now.